core/sql/regress/compGeneral/EXPECTED042

```diff
@@ -1795,8 +1795,8 @@ HQC key=SELECT T . VARCHAR0_UNIQ AS T_VARCHAR0_UNIQ , T . CHAR2_2 AS T_CHAR2_2 ,
 
 Start Time             2015/08/18 15:48:56.468470
 End Time               2015/08/18 15:48:56.471409
-Elapsed Time                      00:00:00.002939
-Compile Time                      00:00:00.002939
```

Comment:
A question to the community: Do we expect these times to match up precisely on each run or is this something that should be filtered in a test run?